TestResultSummaryService/parsers/BenchmarkParser.js

```diff
@@ -152,6 +165,20 @@ class BenchmarkParser extends Parser {
                 benchmarkProduct: curBenchmarkProduct,
                 testData: curTestData
             } );
+            if ( curTestData["metrics"] && curTestData["metrics"].length > 0 ) {
+                for ( let l = 0; l < curTestData["metrics"].length; l++ ) {
+                    const childMetricName = curTestData["metrics"][l]["name"];
+                    const childMean = curTestData["metrics"][l]["mean"];
+                    const childValue = curTestData["metrics"][l]["value"];
+                    childAggData.push({name: childMetricName, value: {mean: childMean, max: math.max(...childValue), min: math.min(...childValue), median: math.median(childValue), stddev: math.std(childValue), CI: BenchmarkMath.confidence_interval(childValue), iteration: 1}});
+                }
+            }
```

Comment:
`iteration` here is the total number raw values used for aggregation information. In parent build it means the total number of its children builds raw values used, while in a child build it means the raw value of itself.

**aggregateInfo**: 
<img width="666" alt="Screen Shot 2019-04-16 at 3 35 46 PM" src="https://user-images.githubusercontent.com/43182493/56238893-97695b00-605d-11e9-808b-a17fc142f1f8.png">

**curTestData** is assigned to **testData** here: 
<img width="669" alt="Screen Shot 2019-04-16 at 3 36 17 PM" src="https://user-images.githubusercontent.com/43182493/56238904-9cc6a580-605d-11e9-908d-dd585efd4cf7.png">

Reply:
It has no usage for the time being, will remove it then.

Reply:
Iteration is useful to have. For each parent build, iteration corresponds to the number of child builds. For each child build, iteration corresponds to the number of tests. For example, a child build for Liberty startup would have 8 iterations (i.e. tests) since we start Liberty multiple times and then take the average for that. We should actually using the length of the array `value` instead of `iteration: 1`.